web/src/main/java/ca/hendriks/bartender/web/ingredient/IngredientsRestController.java

```diff
@@ -28,4 +29,18 @@ public Ingredient addIngredient(@RequestBody Ingredient ingredient) {
         return ingredientRepository.save(ingredient);
     }
 
+    @PutMapping
+    public void updateIngredient(@RequestBody List<String> ingredientChangeList){
+        final int ingredientToBeUpdatedId = ingredientRepository.findByName(ingredientChangeList.get(0)).getId();
+        final IngredientType updatedIngredientType = IngredientType.valueOf(ingredientChangeList.get(3));
+        final Ingredient updatedIngredient = new Ingredient(ingredientToBeUpdatedId, ingredientChangeList.get(2), updatedIngredientType);
+        ingredientRepository.save(updatedIngredient);
+    }
+
+    @DeleteMapping
+    public void deleteIngredient(@RequestBody Object ingredientName){
+        final int ingredientIdToBeDeleted = ingredientRepository.findByName(ingredientName.toString()).getId();
+        ingredientRepository.deleteById(ingredientIdToBeDeleted);
+    }
```

Comment:
good! just FYI, from https://www.rfc-editor.org/rfc/rfc9110.html#name-delete:

> If a DELETE method is successfully applied, the origin server SHOULD send
> 
> a [202 (Accepted)](https://www.rfc-editor.org/rfc/rfc9110.html#status.202) status code if the action will likely succeed but has not yet been enacted, a [204 (No Content)](https://www.rfc-editor.org/rfc/rfc9110.html#status.204) status code if the action has been enacted and no further information is to be supplied, or a [200 (OK)](https://www.rfc-editor.org/rfc/rfc9110.html#status.200) status code if the action has been enacted and the response message includes a representation describing the status.

The method above should probably return 204.

Reply:
Not that big a deal tho, honestly...